packages/connect-voting-disputable/src/helpers/numbers.ts

```diff
@@ -0,0 +1,27 @@
+import { BigNumber, utils } from 'ethers'
+
+export const PCT_DECIMALS = 16 // 100% = 10^18
+
+export const PCT_BASE = BigNumber.from(`100${'0'.repeat(PCT_DECIMALS)}`)
+
+export const bn = (x: string): BigNumber => BigNumber.from(x)
+
+export const formatBn = (number: string | BigNumber, numberDecimals: string | number, formattedDecimals = 2): string => {
+  const formattedNumber = utils.formatUnits(number, numberDecimals)
+  const decimalPosition = formattedNumber.indexOf('.')
+
+  if (decimalPosition === - 1) {
+    return `${formattedNumber}.${'0'.repeat(formattedDecimals)}`
+  }
+
+  const decimals = formattedNumber.substring(decimalPosition + 1)
+  const decimalsLength = decimals.length
+
+  if (decimalsLength <= formattedDecimals) {
+    return `${formattedNumber}${'0'.repeat(formattedDecimals - decimalsLength)}`
+  }
```

Comment:
I'm not super sure about this logic. I think we are missing to round the decimals when `decimalsLength <= formattedDecimals`.

Reply:
Note that scenario is contemplated in the tests, and actually, if the number of decimals is lower than or equal to the number of requested decimals to format, then we don't need to round it, don't you agree?